stl/inc/atomic

```diff
@@ -230,7 +234,11 @@ _NODISCARD inline memory_order _Combine_cas_memory_orders(
 
     _Check_memory_order(_Success);
     _Check_load_memory_order(_Failure);
+#if _HAS_CXX20
+    return _Combined_memory_orders[static_cast<unsigned int>(_Success)][static_cast<unsigned int>(_Failure)];

```

Comment:
It might be reasonable to unconditionally cast; while unnecessary casting is usually undesirable, here we're using enumerators as array indices (which is unusual), so it seems fine to cast even when it's an unscoped enum.

Reply:
Changed to always cast.